src/components/ForgotPassword.js

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import axiosInstance from '../apis/axiosClient';
+
+function ForgotPassword() {
+  const [email, setEmail] = useState('');
+  const [message, setMessage] = useState('');
+  const [error, setError] = useState('');
+
+  const handleSubmit = async () => {
+    try {
+      await axiosInstance.post('/api/v1/auth/forgot_password', { email });
```

Comment:
Our `axiosInstance` already adds in the `/api/v1/` part so that we don't need to repeat it everywhere 😄 

It does that here by setting the `baseURL` to the value of the environment variable `REACT_APP_API_BASE_PATH` https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/8bb8e768ae08fe4a039a860fff7c3f7e5e570c7f/src/apis/axiosClient.js#L5

That environment variable, should be set for everyone locally to `localhost:8000/api/v1` here: https://github.com/Code-the-Dream-School/dd-prac-team4-front/blob/8bb8e768ae08fe4a039a860fff7c3f7e5e570c7f/.env

So right now, this code here is causing us to make a call to `http://localhost:8000/api/v1/api/v1/auth/forgot_password`, which of course is a url that does not exist in the backend

<img width="930" alt="Screenshot 2023-11-08 at 9 41 46 AM" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/69ba6d29-e285-444c-bbf4-26d4e76e1867">

Reply:
I have updated the path 🙈 commit 22471db67c0948ca4a5cd2f7cbc6549a56b571b2

Reply:
Now I see the page to update the password but it throws an error , should be like this at the moment? 

![image](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/88336329/d917fe05-5100-4c89-b508-6cc9ff42288c)

Reply:
What does it show in the network tab for the request made? Or dev console for any error logs?

It works on my end.

Reply:
It is rather weird , when I register , it gives me an error but let me see the page 

<img width="1508" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/88336329/c8bd4aad-b1fb-4c69-8445-443ada4898a6">

The backend terminal 

<img width="939" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/88336329/f75dbbe9-66ad-46d2-a544-ec2458897715">

Reply:
When I try sign in ,it is like this 

<img width="1686" alt="image" src="https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/88336329/82bffa08-c1cb-437e-8d5e-097694fcfa11">